### PR TITLE
Flesh out the README slightly

### PR DESCRIPTION
**Pull request by lol768**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,11 +16,61 @@ cargo install
...
+### Creating pastes from files
+
+Pasting a single file:
+
+```
+$ bins -s gist hello.c
+https://gist.github.com/215883b109a0047fe07f5ee229de6a51
+```
+
+bins supports pasting multiple files, too. With services such as GitHub's [gist](https://gist.github.com), these are natively supported. For services which don't support multiple file pastes, an index paste is created and returned which links to the individual pastes.
```
Comment (jkcclemens) [inline README.md]: "links to individual pastes for each file."

File: README.md [hunk 2]
```diff
@@ -16,11 +16,61 @@ cargo install
...
+```
+
+```
+$ bins -s pastie hello.c goodbye.c
+http://pastie.org/private/x3u62xmyam0i8hiyas8hg
+```
+
+### Specifying visibility options
+
+By default, bins will use the `defaults:private` option from the config file to determine whether or not to create a private paste. The default value of this is `true` - so new pastes will be private for a fresh install. You can override this at the command line:
```
Comment (jkcclemens) [inline README.md]: `defaults.private`

File: README.md [hunk 3]
```diff
@@ -16,11 +16,65 @@ cargo install
...
 ## Usage
 
 To get help, use `bins -h`. bins accepts a list of multiple files, a string, or piped data.
 
-See [asciinema](https://asciinema.org/a/48190) for a demo.
+Take a look at some of the written examples below:
+
+### Examples
+
+#### Creating a paste from stdin
```
Comment (jkcclemens) [inline README.md]: Indents aren't standard. This example is 4, but the rest are 3.

Comment (lol768) [inline README.md]: I've fixed the ones below this line to be consistent with this one (the intention was that they'd all be 4s).

**Comment by jkcclemens** [comment]
It looks good to me if you're done making changes.

**Comment by lol768** [comment]
I am.

**Comment by lol768** [comment]
Thank you!

**Pull request merged**